File: Unmanned delivery service/firebaseRealtimeDatabase/app/src/main/java/com/hada/firebase_realtimedatabase/MainActivity.java
package com.hada.firebase_realtimedatabase;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

import com.google.firebase.FirebaseApp;
import com.google.firebase.database.DataSnapshot;
import com.google.firebase.database.DatabaseError;
import com.google.firebase.database.DatabaseReference;
import com.google.firebase.database.FirebaseDatabase;
import com.google.firebase.database.ValueEventListener;

public class MainActivity extends AppCompatActivity {

    EditText et_login;
    Button bt_login;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        et_login = (EditText) findViewById(R.id.et_login);
        bt_login = (Button) findViewById(R.id.bt_login);

        bt_login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String userId = et_login.getText().toString();
                if (userId.length() < 1){
                    Toast.makeText(getApplicationContext(),"호수를 입력해 주세요",Toast.LENGTH_SHORT);
                }else{
                    Intent myIntent = new Intent(getApplicationContext(), after_loginActivity.class);
                    myIntent.putExtra("id", userId);
                    startActivity(myIntent);
                }
            }
        });
//
    }
}

File: MotionSensor/app/src/main/java/com/example/myapp/MainActivity.kt
package com.example.myapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        startActivity(Intent(this@MainActivity, LoginActivity::class.java))
        finish()

//        // activity_main의 일반 사용자 로그인 버튼을 누르면 activity_login으로 이동
//        UserLoginB.setOnClickListener {
//            startActivity(Intent(this@MainActivity, LoginActivity::class.java))
//        }
//
//        // activity_main의 센터 사용자 로그인 버튼을 누르면 activity_login으로 이동
//        // 일반 사용자/센터 사용자 구분 로그인 아직 안함
//        CenterLoginB.setOnClickListener {
//            startActivity(Intent(this@MainActivity, LoginActivity::class.java))
//        }
//
//        // activity_main의 회원가입 버튼을 누르면 activity_register로 이동
//        RegisterB.setOnClickListener {
//            startActivity(Intent(this@MainActivity, SignUpActivity::class.java))
//        }
//
//        // activity_main의 메인화면(임시)을 누르면 activity_main_list로 이동
//        mainlist.setOnClickListener {
//            startActivity(Intent(this@MainActivity, MainList::class.java))
//        }
    }
}